CCM release notes for v80301 (#2772)
harness/developer-hub: release-notes/cloud-cost-management.md

:::

## Latest - August 02, 2023, version 80301

```mdx-code-block
<Tabs>
  <TabItem value="What's new">
```
* AWS AutoStopping proxy enhancement (CCM-13497)

  You can now select the subnet ID from the dropdown list for AWS AutoStopping proxy creation. 

    <docimage path={require('./static/ccm-subnet-proxy.png')} width="60%" height="60%" title="Click to view full size image" />

> * **Perspective Preferences** enhancement

Author: Added the Jira ticket

  Perspective preferences provide you the flexibility to control which cost factors are considered in your billing and CUR (Cost and Usage Report) reports within your perspective. You can now include cost factors such as discounts, taxes, refunds, etc. For more information, go to [Perspective Preferences](../docs/cloud-cost-management/3-use-ccm-cost-reporting/1-ccm-perspectives/perspective-preferences.md).

 
:::important note
 The current configurations for **Show others** and **Show unallocated cost in clusters** are preserved. This means that though the default settings have these preferences set to false, any **existing perspective** with these preferences set to true will retain their current state and not be overridden.
:::

* Improved UI handling during the AutoStopping rule creation process (CCM-13527)

>   The Load Balancer or AutoStopping Proxy selection screen has been enhanced to include an additional API that retrieves information about proxies created previously in shared VPCs. Now users can use proxy created in shared VPC across projects and connectors.

Author: Rephrased:
The page on which users select either a load balancer or an AutoStopping Proxy has been enhanced to include an additional API that retrieves information about proxies created previously in shared VPCs. Now users can use a proxy created in a shared VPC across projects and connectors.

* **Overview** page enhancements (CCM-13326)

  - Added hover state to the pie chart 

>   - Forecast trend from the widget is hidden.

Author: The forecast trend in the widget is removed.

  - Added forecast time period in the forecast cost widget.
  - Changed the heading of the cloud cost widget from `Top AWS accounts` to `Top 10 AWS accounts`.

    <docimage path={require('./static/ccm-overview-1.png')} width="60%" height="60%" title="Click to view full size image" />

>     <docimage path={require('./static/ccm-overview-2.png')} width="60%" height="60%" title="Click to view full size image" />

Reviewer: This screenshot shows an employee's name.

Author: Blurred the name and account ID

  </TabItem>
  <TabItem value="Fixed issues">
```
* Previously, configuring both the redirect URL and target port for redirection while creating a redirect-based AutoStopping rule led to an error. (CCM-13475)

>   This issue has been resolved by modifying the validation process to ensure that when the redirect URL is defined, the system will now check if the target port is greater than 0. The combination of both redirect URL and target port is not allowed as it is an invalid configuration. However, for ALBs only redirect URLs are allowed.

Reviewer: ```suggestion
  This issue has been resolved by modifying the validation process. Now, if the redirect URL is defined, the validation process checks whether the target port is greater than 0. Specifying both redirect URL and target port is not allowed as it is an invalid configuration. However, for ALBs only redirect URLs are allowed.
```
"However, for ALBs only redirect URLs are allowed" is missing a comma, but I don't know where to add it. "However, for ALBs, only redirect URLs are allowed" or "However, for ALBs only, redirect URLs are allowed"? Also maybe lean towards the active voice here. 

(Also, please check whether I've changed the meaning with my edits. Thank you.)

Author: Thanks so much, @deepakbolangady . Valid comments. I drafted these from the comments and descriptions in the tickets and got them technically reviewed. Except for the one that I have rephrased, I have accepted all your suggestions.

> * Users encountered an issue with the Kubernetes AutoStopping rule creation process as they were not able to validate the YAML on the UI. (CCM-13459)

Reviewer: Is the issue described in this release note? It reads as if users faced some issue and were therefore unable to validate YAML files. Or, is this accurate:
```suggestion
* Users were unable to validate their YAML files when creating a Kubernetes AutoStopping rule. (CCM-13459)
```
